src/test/java/test/IngressTest.java

```diff
@@ -210,4 +210,56 @@ void jira_ingress_path_default(Product product) throws Exception {
         }
     }
 
+    @ParameterizedTest
+    @EnumSource(value = Product.class, names = "confluence")
+    void confluence_ingress_path_contextPath(Product product) throws Exception {
```

Comment:
We should add one more unit test where `confluence.service.contextPath` is not provided, but `ingress.path` is provided. All 4 combination should be tested.

Reply:
I think this test is not really needed as based on the logic always path has more priority and if a test is existed and passed with context and path, the result with another test with path and no context would be identical. (if path is existed the workflow does not consider context at all, while the value is provided or not)

Reply:
Anyhow, having an extra test doesn't hurt - so I added an extra test for that scenario.